### PR TITLE
fix: author bar styles updated 

### PR DESCRIPTION
**Pull request by kirillkurko**
### Describe what this PR does

Author bar is not a children of card image anymore, so we need to style it manually to match the image (`border-radius` and `margin-top`). Also while looking for this component found some commented lines, I don't like to keep commented lines, so removed it 

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [x] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-243 #done

### CODE REVIEW

File: packages/shared/src/components/Feed.tsx [hunk 1]
```diff
@@ -166,17 +166,9 @@ export default function Feed<T>({
     return <></>;
   }
 
-  // const {
```
Comment (rebelchris) [inline packages/shared/src/components/Feed.tsx:169]: Can you please revert this for now.
We need it in the code for when we urgently need to revert back, it's still unclear if this removal has a big impact.

Perhaps you can add a comment that explains this, as I can see why you removed it.

**Review by rebelchris — CHANGES_REQUESTED** [review]
Perfect fix, but please revert the removal.
It's a bit weird, but it's because we are unsure so please feel free to add a comment explaining it should be there.

**Comment by kirillkurko** [comment]
@rebelchris I reverted removed commented code, but I have a question - how revert commit message should look? Because default message from WebStrorm is not matching our conventional commits guideline

**Comment by rebelchris** [comment]
@kirillkurko Thanks, automated commit messages like this are fine.
They are a edge-case anyway :)

Feel free to merge this one.

**Comment by kirillkurko** [comment]
> @kirillkurko Thanks, automated commit messages like this are fine. They are a edge-case anyway :)
> 
> Feel free to merge this one.

Can you approve this one, changes are still requested from you

**Review by rebelchris — APPROVED** [review]

**Pull request merged**